lib/house.rb

```diff
@@ -0,0 +1,42 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
```

Comment:
this isn't quite right-- the price is a string when it gets passed in, so we'd need to convert 

`@price = price.to_i` to get this to work the way the interaction pattern expects